### ISSUE TITLE
Turntable doesn't rotate

### ISSUE THREAD

**Issue by applekillerbyjames** [morpheusthewhite/spicetify-themes#416]
**Describe the bug**
Despite having turntable theme and extensions added, turntable rotation doesn't work.

**To Reproduce**
1. Go to Spotify
2. Click on Play button
3. Scroll down to Player area
4. Album art not rotating

**Expected behavior**
Album should rotate

**Screenshots**

**Logs**
Add logs from console. To do that
1. Run `spicetify enable-devtools` in terminal
2. Spotify will be restarted
3. Hit <kbd>Ctrl + Shift + I</kbd> to open DevTools window
4. Navigate to tab Console
5. Copy console window content.

```console
(Please paste here console logs or attach a screenshot)
```

**Specifics (please complete the following information):**
 - OS: [e.g. iOS] macOS 12
 - Spotify version [e.g. 1.1.10.546.ge08ef575] 1.1.62.583.gdac868ed-a
 - Spicetify version [e.g. 1.1.0] 2.3.0
 - Theme name Turntable

**Comment by grasonchan** [comment]
I don't know what is "Scroll down to Player area". Do you mean open the Full App Display mode?

"turntable rotation doesn't work", it is not work or not normal? What happens when click the play button again? Can you upload some gifs or videos?

It's normal on my macOS Big Sur, but have question when i tested on Windows 10, and maybe i fixed it now but not commit yet. Maybe you can go to [my repository](https://github.com/grasonchan/spotify-spice) to get and test it later.

**Comment by applekillerbyjames** [comment]
It doesn't work without FullAppDisplay, which you have mentioned (my issue) in the spicetify github. However, I expected the rotation to just work in the normal player, with the album art at the bottom left.

**Comment by grasonchan** [comment]
> It doesn't work without FullAppDisplay, which you have mentioned (my issue) in the spicetify github. However, I expected the rotation to just work in the normal player, with the album art at the bottom left.

Yep, it does not support the album art at the Now Playing Bar. Actually, i tried to support it, but the effect is not good.

**Comment by applekillerbyjames** [comment]
having now seen FullAppDisplay, it only rotates when paused.

**Comment by grasonchan** [comment]
Did you follow the [instruction](https://github.com/morpheusthewhite/spicetify-themes/tree/v2/Turntable)?

**Comment by grasonchan** [comment]
@applekillerbyjames I fixed some bugs, maybe it can correctly control the rotation of the turntable now. Can you test it and give me some feedback? Remember to follow the [installtion instruction](https://github.com/morpheusthewhite/spicetify-themes/tree/v2/Turntable).

**Comment by applekillerbyjames** [comment]
I cannot use spicetify, so I cant test

**Comment by grasonchan** [comment]
> I cannot use spicetify, so I cant test

Try to reinstall Spotify and Spicetify.

**Comment by morpheusthewhite** [comment]
@applekillerbyjames try testing it now with the latest spicetify version
